src/map/pc.cpp

```diff
@@ -9033,7 +9033,7 @@ void pc_skillup(map_session_data *sd,uint16 skill_id)
 			lv = sd->status.skill[idx].lv;
 			range = skill_get_range2(&sd->bl, skill_id, lv, false);
 			upgradable = (lv < skill_tree_get_max(sd->status.skill[idx].id, sd->status.class_)) ? 1 : 0;
```

Comment:
```suggestion
			bool upgradable = (lv < skill_tree_get_max(sd->status.skill[idx].id, sd->status.class_)) ? true : false;
```
remove upgradable from `int lv, range, upgradable;` above

Reply:
changed